docs/intro/tutorial03.txt

```diff
@@ -133,7 +133,7 @@ The colon (``:``) separates the converter and pattern name.
 Write views that actually do something
 ======================================
 
-Each view is responsible for doing one of two things: returning an
+Each view is responsible for doing one of two things: returning a
```

Comment:
This is not a typo, check out [docs](https://docs.djangoproject.com/en/5.0/internals/contributing/writing-documentation/#commonly-used-terms).